### PR TITLE
fix(forms) org scope input accounts for .form-control class

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

**Comment by aredridel** [comment]
screen shot 2016-05-12 at 18 36 06

File: src/pivotal-ui/components/forms/forms.scss [hunk 1]
```diff
@@ -1387,14 +1387,14 @@ An input for an npm scope, prompting the user with an @ sign.
...
-  padding-left: calc(1.5em); // Not exact, but the required distance _does_ depend on typeface. This assumes @ and the line height are in a decent ratio, which seems a reasonable approximation.
+  padding-left: 1.5em; // Not exact, but the required distance _does_ depend on typeface. This assumes @ and the line height are in a decent ratio, which seems a reasonable approximation.
 }
 .scope-input::before {
   position: absolute;
   content: "@";
   speak: none;
-  padding: 3px;
-  padding-left: 0.3em;
+  z-index: 1;
```
Comment (stubbornella) [inline src/pivotal-ui/components/forms/forms.scss]: if you made it :after you wouldn't need to change z-index (I think)

Comment (aredridel) [inline src/pivotal-ui/components/forms/forms.scss]: Oh good call.

**Comment by jefflembeck** [comment]
If we're using the search input as a guideline for this, for consistency's sake, the spacing is just a tiny bit off

screen shot 2016-05-16 at 7 25 37 am

vs.

screen shot 2016-05-16 at 7 25 50 am

**Comment by aredridel** [comment]
Oh good call. Hmm. @jerrygabra?

**Comment by gabrrra** [comment]
looks good @aredridel

**Pull request merged**